src/github3/checks.py

```diff
@@ -0,0 +1,279 @@
+# -*- coding: utf-8 -*-
+"""This module contains all the classes relating to Checks."""
+from __future__ import unicode_literals
+
+from json import dumps
+
+from . import apps
+from . import decorators
+from . import models
+
+
+class CheckSuite(models.GitHubCore):
+    """The :class:`CheckSuite <CheckSuite>` object.
+
+    .. versionadded:: 1.3.0
+
+    Please see GitHub's `CheckSuite Documentation`_ for more information.
+
+    .. attribute:: status
+
+        The status of the Check Suite
+
+    .. attribute:: conclusion
+
+        The highest priority check run conclusion. If it has not completed this
+        will be None
+
+    .. attribute:: head_sha
+
+        The sha of the commit at the head of the branch checked
+
+    .. attribute:: head_branch
+
+        The branch checked
+
+    .. attribute:: before
+
+        The sha of the target branch before the change
```

Comment:
Is "target branch" here the same as `head_branch`? Should we note that in here, something like

```rst
The sha of the target branch (also referred to as, ``head_branch``) before the change.
```

Reply:
No, I think it's different. I believe this represents the sha of the branch the PR would like to merge to. If the PR is on branch `bar` and the target is `master`, then `head_branch` would be `bar` and `before` is the `HEAD` of `master` at the time the check ran. Since `master` may get updated after tests run it's a useful bit of data to validate whether or not the run results are still valid.  Clearly I need to clarify here 😁